scripts/test_execution.json

```diff
@@ -3,34 +3,106 @@
 
     "VMMap":
     {
-        "TK5-3WP08R0904":
+        "TK5-3WP08R0904_WS2019_1":
         [
             {
-                "Name": "vm1"
+                "Name": "vm1_ws2019"
             }
         ],
-        "TK5-3WP08R0904_2":
+        "TK5-3WP08R0904_WS2019_2":
         [
             {
-                "Name": "vm2"
+                "Name": "vm2_ws2019"
             }
         ],
-        "TK5-3WP08R0907":
+        "TK5-3WP08R0907_WS2019_1":
         [
             {
-                "Name": "vm1"
+                "Name": "vm1_ws2019"
             }
         ],
-        "TK5-3WP08R0907_2":
+        "TK5-3WP08R0907_WS2019_2":
         [
             {
-                "Name": "vm2"
+                "Name": "vm2_ws2019"
             }
         ],
-        "TK5-3WP07R0703_1":
+        "TK5-3WP07R0703_WS2019_1":
         [
             {
-                "Name": "vm_perf"
+                "Name": "vm1_ws2019"
+            }
+        ],
+        "TK5-3WP07R0703_WS2019_2":
+        [
+            {
+                "Name": "vm2_ws2019"
+            }
+        ],
+        "TK5-3WP07R0703_WS2019_3":
+        [
+            {
+                "Name": "vm3_ws2019"
+            }
+        ],
+        "TK5-3WP07R0703_WS2019_4":
+        [
+            {
+                "Name": "vm4_ws2019"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_1":
+        [
+            {
+                "Name": "vm1_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_2":
+        [
+            {
+                "Name": "vm2_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_3":
+        [
+            {
+                "Name": "vm3_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_4":
+        [
+            {
+                "Name": "vm4_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_5":
+        [
+            {
+                "Name": "vm5_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_6":
+        [
+            {
+                "Name": "vm6_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_7":
+        [
+            {
+                "Name": "vm7_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_8":
+        [
+            {
+                "Name": "vm8_ws2022"
+            }
+        ],
+        "TK5-3WP07R0703_WS2022_PERF_1":
+        [
+            {
+                "Name": "vm_perf_ws2022"
```

Comment:
should this be vm1_perf to match the other cases?

Reply:
Unless you feel strongly about this one, I'd just leave as is since there's only one perf runner.
As for correctness, it's fine.